docs/how-to/deploy_sdcore_upf_machine.md

```diff
@@ -0,0 +1,40 @@
+# Deploy SD-Core User Plane Machine
+
+This guide covers how to deploy the User Plane Function (UPF) as a machine charm.
+
+## Requirements
+
+- A host with a CPU supporting AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+- [Juju][Juju] controller bootstrapped to a LXD cluster
+- A machine added to the Juju controller
+-  [Terraform][Terraform] installed
+- Git
+
+## Deploy
+
+Get Charmed Aether SD-Core Terraform UPF Machine module by cloning the [Charmed Aether SD-Core UPF module][Charmed Aether SD-Core UPF modules] Git repository. Inside the `terraform` directory, create a `terraform.tfvars` file to set the name of Juju model and machine number for the deployment:
+
+```console
+git clone https://github.com/canonical/sdcore-upf-operator.git
+cd terraform
+
+cat << EOF > terraform.tfvars
+machine_number = 0
+model_name = "user-plane"
```

Comment:
We need to tell user to add config options to match with environment.
As a example:

config = {
  core-interface-name = "enp6s0"
  access-interface-name = "enp7s0"
}

Reply:
+1 
And it's not just about the network interfaces. The rest of the networking params needs to be here as well (unless you're super lucky to be using the addressing matching the defaults :) ).

Reply:
I agree that we should probably list most of the settings, even if we just repeat the defaults, this will make it easier to follow and tweak correctly.

Reply:
@Gmerold @gatici I also addressed this comment.